apps/android/app/src/main/java/chat/simplex/app/views/chat/SendMsgView.kt

```diff
@@ -150,10 +86,216 @@ fun SendMsgView(
           )
         }
       }
+      if (showVoiceButton) {
```

Comment:
why is it not in `if (!showVoiceButton)` else branch, and instead on a higher level?

Reply:
Previously it was better to do like this but with a new behaviour and UI, it can be placed in if statement, yes.